website/addons/base/views.py

```diff
@@ -27,10 +28,9 @@
 from framework.transactions.handlers import no_auto_transaction
 from website import mails
 from website import settings
-from website.addons.base import StorageAddonBase
 from website.addons.base import exceptions
 from website.addons.base import signals as file_signals
-from website.files.models import FileNode, StoredFileNode, TrashedFileNode
+from osf.models.files import FileNode, StoredFileNode, TrashedFileNode
```

Comment:
Would you import `FileNode` (and anything else defined in `osf.models.files` and not explicitly imported into `osf.models`) into `osf.models` and then import all of these from there? I'm trying to keep the pattern where everything comes from `osf.models` but nothing is actually in `osf.models`.